pages.de/android/screencap.md/screencap.md

```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Nimmt ein Bildschirmfoto von einem Handy-Display auf
+> Dieser Befehl kann nur über die 'adb shell' benutzt werden
+> Für weitere Informationen: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Ein Bildschirmfoto aufnehmen:
```

Comment:
As [stated in our Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood), example descriptions are phrased in an imperative mood.

```suggestion
- Nehme ein Bildschirmfoto auf:
```

Reply:
should be "Nimm ein Bildschirmfoto auf" then.

Reply:
@gutjuri `de/linux/arecord` already uses "Nehme [...] auf" and I couldn't find anything with "Nimm", so I thought we might wanna standardize on that